### DISCUSSION TITLE
Generation with extra columns in join table

### DISCUSSION THREAD

**Discussion by agis** [aarondl/sqlboiler#958] [Q&A] [answered]
Hello!

We have a many-to-many relation ship with a join table and its generated models are working fine. However, we now need to add extra columns in that join table. We've noticed that SQLBoiler gets confused when there are extra columns and the generated models don't look correct.

Is there a way to overcome this issue? Thanks in advance!

**Comment by agis** [comment, marked as answer]
I've just noticed https://github.com/volatiletech/sqlboiler/issues/81, so I assume that having the same helper relation ship methods in the original models while also having extra columns in their join table is not possible?

**Reply by aarondl** [reply]
Pretty much #81 describes it perfectly yes. As soon as you add columns you lose those helpers because in its eyes it becomes just a regular old table currently and so it can't rightly add the transitive helpers.
